test/junit/scala/collection/immutable/LazyListLazinessTest.scala

```diff
@@ -448,7 +448,7 @@ class LazyListLazinessTest {
 
   @Test
   def sizeCompare_properlyLazy(): Unit = {
-    for (factory <- LazyList :: Vector :: Nil) {
+    for (factory <- List[SeqFactory[Seq]](LazyList, Vector)) {
```

Comment:
@lrytz this is what was murdering compile times?!

Reply:
Yes, see Jason's analysis https://github.com/scala/bug/issues/11249